R/initModel.R

```diff
@@ -39,6 +40,7 @@ initModel <- function(config = NULL,
                       restart = NULL,
                       sendToSlurm = NULL,
                       slurmQOS = "default",
```

Comment:
I actually had it like this at some point. And it took me a while to remember why I changed it again ... At the beginning of ```setSlurmConfig``` it is checked whether a valid QOS was passed. This check works just as before with ```slurmQOS=="default"```, but would need to be adapted otherwise (which is of course possible). Also for me it is sort of clearer then that the function decides on the right QOS depending whether ```tasks32``` is true or not.

On the other hand, setting the default to ```NULL``` would allow to pass ```default``` by the function call in order to overwrite a setting in the config. So I'm a bit undecided now.